packages/colorpickers/src/elements/ColorpickerDialog/index.tsx

```diff
@@ -60,6 +60,10 @@ export interface IColorpickerDialogProps extends IColorpickerProps {
    * Applies inset `box-shadow` styling on focus
    */
   focusInset?: boolean;
+  /**
+   * Passes HTML attributes to the color dialog button element
+   */
+  triggerProps?: HTMLAttributes<HTMLButtonElement>;
```

Comment:
[nit] I'm leaning toward naming this `buttonProps` as these props are _not_ applied when a generic child trigger is in use.